MediaBrowser.Api/Movies/MoviesService.cs

```diff
@@ -228,17 +229,14 @@ private IEnumerable<RecommendationDto> GetRecommendationCategories(User user, st
             {
                 var allEmpty = true;
 
-                foreach (var category in categoryTypes)
+                foreach (var category in categoryTypes.Where(category => category.MoveNext()))
```

Comment:
Was clearer inside the foreach statement imo.

Reply:
Fixed